src/shared/globals/globals.js

```diff
@@ -10,7 +10,25 @@ import { getDesktopMediaSource } from '../../talk/renderer/screensharing/screens
 import { dialogs } from './OC/dialogs.js'
 import { MimeTypeList } from './OC/mimetype.js'
 
-const webroot = appData.serverUrl ? new URL(appData.serverUrl).pathname : ''
+let webroot
+/**
+ * Compute and cache webroot value based on server URL
+ *
+ * @return {string} webroot value
+ */
+function getWebroot() {
+	if (webroot !== undefined) {
+		return webroot
+	}
+
+	try {
+		webroot = appData.serverUrl ? new URL(appData.serverUrl).pathname.replace(/\/$/, '') : ''
+	} catch (error) {
+		console.error('Failed to parse server URL:', error)
+		webroot = ''
+	}
+	return webroot
+}
```

Comment:
This module already has `initGlobals` function to explicitly init everything when it's ready (where `_oc_webroot` is defined at the moment.

Let's just set `webroot` there instead of implementing a getter with memoization for it.